scripts/debug_commands/cat_pregnancy.py

```diff
@@ -0,0 +1,147 @@
+            kits_amt = game.clan.pregnancy_data[current_cat.ID]["amount"]
+
+        severtity = args[3] if len(args) > 3 else None
+        if not severtity or severtity in ("same" or "" or "s"):
+            severtity = current_cat.injuries["pregnant"]["severity"]
+
+        second_parent = args[4] if len(args) > 4 else None
+        if not second_parent or second_parent in ("same" or "" or "s"):
+            second_parent = game.clan.pregnancy_data[current_cat.ID]["second_parent"]
+        
+        second_parent_cat = get_cat_from_name_or_id(second_parent) if second_parent else None
+        second_parent_repr = f"{second_parent_cat.name} ({second_parent_cat.ID})" if second_parent_cat else "None"
+        if "pregnant" in current_cat.injuries:
+            game.clan.pregnancy_data[current_cat.ID]["moons"] = int(moons_amt)
+            game.clan.pregnancy_data[current_cat.ID]["amount"] = int(kits_amt)
+            current_cat.injuries["pregnant"]["severity"] = severtity
+            game.clan.pregnancy_data[current_cat.ID]["second_parent"] = second_parent
+            add_output_line_to_log(f"Successfully edited pregnancy of {current_cat.name} ({current_cat.ID}), new pregnancy data: ")
+            add_multiple_lines_to_log(f"""Moons: {moons_amt}
+                                        Amount of Kits: {kits_amt}
+                                        Severity: {severtity}
+                                        Second Parent: {second_parent_repr}""")
+        else:
+            add_output_line_to_log("Specified cat is not pregnant")
+
+class ViewPregnancyCommand(Command):
+    name = "view"
+    description = "View the stats a pregnancy"
+    aliases = ["v"]
+
+    usage = "<cat id>"
+
+    def callback(self, args: List[str]):
+        if len(args) == 0:
+            add_output_line_to_log("Please specify the name/id of the cat to edit the pregnancy of.")
+            return
+        cat = get_cat_from_name_or_id(args[0])
+     
+        second_parent_cat = get_cat_from_name_or_id(game.clan.pregnancy_data[cat.ID]["second_parent"]) if game.clan.pregnancy_data[cat.ID]["second_parent"] else None
+        second_parent_repr = f"{second_parent_cat.name} ({second_parent_cat.ID})" if second_parent_cat else "None"
+        if "pregnant" in cat.injuries:
+            add_multiple_lines_to_log(f"""Cat: {cat.name} ({cat.ID})
+                                        Moons: {game.clan.pregnancy_data[cat.ID]["moons"]}
+                                        Amount of Kits: {game.clan.pregnancy_data[cat.ID]["amount"]}
+                                        Severity: {cat.injuries["pregnant"]["severity"]}
+                                        Second Parent: {second_parent_repr}""")
+        else:
+            add_output_line_to_log("Specified cat is not pregnant")
+
+class PregnanciesCommand(Command):
+    name = "pregnancies"
+    description = "Manage Cat Pregnancies"
+    aliases = ["preg", "p"]
```

Comment:
```suggestion
    aliases = ["preg", "p", "pregnancy"]
```
Purely a suggestion. Otherwise, looks good to me!